<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="world_config_path" default="$(find semantics_manager)/config/config.yaml"/>
  <arg name="output" default="screen"/>

  <include file="$(find asoom)/launch/asoom.launch" pass_all_args="True"/>
</launch>
